stories/example-screen-staff-list.stories.js

```diff
@@ -27,150 +17,1023 @@ export default {
+                  </div>
+                </div>
+                <div class="col-md-6 col-lg-4 d-flex justify-content-end flex-wrap my-1">
+                  <button type="button" class="btn btn-outline-secondary ms-2 fs-6">
+                    <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+                      <use xlink:href="static/media/node_modules/@symplicity/tabler-icons/symplicity/tabler-sprite-nostroke.svg#tabler-search" />
+                    </svg>Apply Search
+                  </button>
+                  <button type="button" class="btn btn-outline-secondary ms-2 fs-6">
+                    <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+                      <use xlink:href="static/media/node_modules/@symplicity/tabler-icons/symplicity/tabler-sprite-nostroke.svg#tabler-adjustments-horizontal" />
+                    </svg>More Filters
+                  </button>
+                </div>
+              </form>
+            </div>
+            <div class="d-flex align-items-center justify-content-between flex-wrap py-3">
+              <div class="d-flex align-items-center justify-content-center">
+                <select class="form-select fs-6" aria-label="Default select example">
+                  <option selected>Batch Options</option>
+                  <option value="1">One</option>
+                  <option value="2">Two</option>
+                  <option value="3">Three</option>
+                </select>
+                <span class="fs-6 text-muted ms-3" style="min-width: 120px;">1 item selected</span>
+              </div>
+              <div class="d-flex align-items-center flex-wrap">
+                <div class="d-flex">
+                  <button type="button" class="btn btn-action btn-action-icon-only border-light">
+                    <svg class="icon-lg-font-size mt-n1">
+                      <use xlink:href="${iconSprite}#tabler-chevron-left" />
+                      <span class="visually-hidden">Previous</span>
+                    </svg>
+                  </button>
+                </div>
+                <div class="d-flex mx-0">
+                  <select class="form-select shadow-none border-light fs-6" aria-label="Default select example">
+                    <option value="2">Page 2</option>
+                    <option value="3">Page 3</option>
+                    <option value="4">Page 4</option>
+                  </select>
+                </div>
+                <div class="d-flex">
+                  <button type="button" class="btn btn-action btn-action-icon-only border-light">
+                    <svg class="icon-lg-font-size mt-n1">
+                      <use xlink:href="${iconSprite}#tabler-chevron-right" />
+                      <span class="visually-hidden">Next</span>
+                    </svg>
+                  </button>
+                </div>
+              </div>
+              <div class="w-100 overflow-scroll p-0 my-2 border border-light">
```

Comment:
Change `overflow-scroll` to `overflow-auto` here so that scrollbars don't show in Chrome unless there is enough content to make them show.